### ISSUE TITLE
SQLite silently drops statements in multi-statement query

### ISSUE THREAD

**Issue by spiffytech** [oven-sh/bun#3283]
### What version of Bun is running?

0.6.8

### What platform is your computer?

Linux 6.3.5-200.fc38.x86_64 x86_64 unknown

### What steps can reproduce the bug?

If you execute a query that contains multiple SQL statements, Bun will run the first statement, but silently fails to execute the rest.

```ts
import { Database } from "bun:sqlite";

const db = new Database(":memory:");
db.query(`
  pragma user_version = 1;
  pragma user_version = 2;
`).run();

console.log(db.query("pragma user_version").get());
```

### What is the expected behavior?

Both statements are executed. Or an error is thrown.

### What do you see instead?

Only the first statement is executed: `user_version` is set to `1`.

### Additional information

`better-sqlite3` runs both statements when using `db.exec()`, and throws an error when using `db.prepare()`.

**Comment by Jarred-Sumner** [comment]
Hm, implies there's some sqlite3 status value we are not handling correctly in `run`:

https://github.com/oven-sh/bun/blob/main/src/bun.js/bindings/sqlite/JSSQLStatement.cpp#L1458-L1476

**Comment by Jarred-Sumner** [comment]
maybe it needs to be SQLITE_DONE instead of SQLITE_OK

**Comment by raashidanwar** [comment]
Hi @Jarred-Sumner, I want to pick this up.

**Comment by raashidanwar** [comment]
> maybe it needs to be SQLITE_DONE instead of SQLITE_OK

The problem is with the status here 👉 https://github.com/oven-sh/bun/blob/main/src/bun.js/bindings/sqlite/JSSQLStatement.cpp#LL1467C1-L1467C1. It suppose to be 100 but somehow it coming as 101 and because of that the next query is not getting executed.

**Comment by Electroid** [comment]
This was fixed in #8451, though you'll need to use `run()` instead of `query()`, like so:

```js
import { Database } from "bun:sqlite";

const db = new Database(":memory:");
db.run(`
  pragma user_version = 1;
  pragma user_version = 2;
`);

console.log(db.query("pragma user_version").get());
```
